myfirstmarkdown

```diff
@@ -0,0 +1,9 @@
+This is First Github Markdown File and i have always dreamed of learning the version control with git and here i am , and my main goal is to be a software employee and there are many other courses i have been learning apart from the classroom , from coursera,udemy.and my faviourate hobby is to learn new things by internet.
+
+Here i have learned how to create a new repo and pull a rewuest and make changes , and repo is a main location where we store all our packages required into
+
```

Comment:
**_### The geeky Stuff is learning the new terminolgy of Github 
like branching 
forking
cloning of a repo
commiting , 
pull and push requests
and there are quite understandable and tricky notes like exploring market place and eploring others repositories_**